docs/sample-config.ini

```diff
@@ -163,6 +163,11 @@
 #
 # activityNotifyCommand = /path/to/notify
 
+# Notification script control: select the notifier payload version.
+# This affects how the activityNotifyCommand is invoked.
```

Comment:
Can this documentation be updated to list the currently-acceptable values?

Reply:
Resolved by ba908325.